e2etests/bats-tests.sh

```diff
@@ -0,0 +1,527 @@
+#!/usr/bin/env bats
+
+load bats-support-clone
+load test_helper/bats-support/load
+load test_helper/redhatcop-bats-library/src/error-handling.bash
+
+@test "access-to-create-pods" {
+  tmp="tests/checks/access-to-create-pods.yml"
+  cmd="${KUBE_LINTER_BIN} lint --include access-to-create-pods --do-not-auto-add-defaults ${tmp}"
+  run ${cmd}
+
+  print_info "${status}" "${output}" "${cmd}" "${tmp}"
+  [ "$status" -eq 1 ]
+  [[ "${lines[1]}" == "tests/checks/access-to-create-pods.yml: (object: namespace-dev/role-binding1 rbac.authorization.k8s.io/v1, Kind=RoleBinding) binding to \"role1\" role that has [create] access to [pods] (check: access-to-create-pods, remediation: Where possible, remove create access to pod objects in the cluster.)" ]]
```

Comment:
I think we should use the JSON format, and use `jq` to do the assertions. Otherwise, if we change the output format, updating the tests will be a pain. The tests would also be more readable that way.

Reply:
pushed a commit (https://github.com/stackrox/kube-linter/pull/238/commits/560d5615c80ba0c651405a01bff485be66adfe3c) - do you mean something like this?

Reply:
Yes, exactly.